tests/e2e/longevity/test_stage0.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from ocs_ci.framework.testlib import E2ETest, ignore_leftovers
+from ocs_ci.ocs.longevity import Longevity
+
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+class TestLongevity(E2ETest):
+    """
+    Test class for Longevity: Stage-0
+    """
+
+    def test_stage_0(self, project_factory):
```

Comment:
@PrasadDesala Would like to know if we are collecting some data from stage0 test and using it in stage1 tests or in other stages

Reply:
Yes @ramkiperiy I may have to use the resources data that I created in stage 0 in other stages. The implementation for this will be done in a separate PR